raiden/storage/sqlite.py

```diff
@@ -1474,6 +1484,18 @@ def update_light_client_protocol_message_set_signed_data(self, payment_id, msg_o
             last_id = cursor.lastrowid
         return last_id
 
+    def update_onchain_light_client_protocol_message_set_signed_transaction(self,
```

Comment:
this says _set transaction_ but it receives a signed _message_

this isn't structurally equivalent to the other call :man_shrugging:

Reply:
Basically this 2 methods are totally different in terms of business, are for 2 different use cases (payment related and not) so i guess the guys think those names are ok, anyway again, i think we should stablish some sort of criteria here because my first suggestion wasn't good neither.

Reply:
I had not noticed this before, but indeed, if we receive a signed transaction, the typing won't work, will it?